### PR TITLE
[libepoxy] update to 1.5.10

### LINKED ISSUE #27395: [libepoxy] update to 1.5.10
Library name: libepoxy

New version number: 1.5.10

Other information that may be useful (release notes, etc...)
https://github.com/anholt/libepoxy/releases/tag/1.5.10

### PR DESCRIPTION
**Pull request by SchaichAlonso**
**Describe the pull request**

- #### What does your PR fix?
  Fixes #27395 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  tripplets unchanged

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  `Yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/libepoxy/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "libepoxy",
-  "version-semver": "1.5.9",
-  "port-version": 1,
+  "version-semver": "1.5.10",
   "description": "Epoxy is a library for handling OpenGL function pointer management for you",
```
Comment (JonLiu1993) [inline ports/libepoxy/vcpkg.json:4]: @SchaichAlonso, Please add license in vcpkg.json.

File: ports/libepoxy/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "libepoxy",
-  "version-semver": "1.5.9",
-  "port-version": 1,
+  "version-semver": "1.5.10",
```
Comment (JonLiu1993) [inline ports/libepoxy/vcpkg.json:3]: @SchaichAlonso, I looked into Upstream and found no evidence that it follows version-semver, I suggest we use version.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

File: ports/libepoxy/vcpkg.json [hunk 3]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "libepoxy",
-  "version-semver": "1.5.9",
-  "port-version": 1,
+  "version": "1.5.10",
   "description": "Epoxy is a library for handling OpenGL function pointer management for you",
   "homepage": "https://github.com/anholt/libepoxy",
+  "license": "MIT",
```
Comment (BillyONeal) [inline ports/libepoxy/vcpkg.json:6]: https://github.com/anholt/libepoxy/blob/master/COPYING ✅

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**